LEGO1/legogamestate.cpp

```diff
@@ -255,14 +261,51 @@ void LegoGameState::RegisterState(LegoState* p_state)
 	m_stateArray[targetIndex] = p_state;
 }
 
-// STUB: LEGO1 0x1003cdd0
-void LegoGameState::SerializeScoreHistory(MxS16)
+void LegoGameState::WriteScoreHistory(MxU16*)
+{
+}
+
+// STUB: LEGO1 0x1003ccf0
+void FUN_1003ccf0(MxU16*, LegoFileStream&)
```

Comment:
Since `LegoGameState::WriteScoreHistory` is part of `LegoGameState`, and both are called with `m_unk0xa6`, shouldn't both be part of the `LegoGameState` class?

Reply:
It doesn't seem like ```FUN_1003ccf0``` is part of the LegoGameState
![image](https://github.com/isledecomp/isle/assets/106913236/5739a26d-5b61-4a3e-8132-0e7e6ef20e3c)

Reply:
Yes, but why do you think `WriteScoreHistory` is? Both functions are being called the same way:

![image](https://github.com/isledecomp/isle/assets/1135351/c6b1eeee-9360-46a9-8b7a-92d087ad9d81)

Reply:
Oh, I see what you mean, sorry